metashare/api/models.py

```diff
@@ -179,6 +182,28 @@ class Meta:
         ordering = ("name",)
         unique_together = (("repo_owner", "repo_name"),)
 
+    def get_a_matching_user(self):
+        github_repository = GitHubRepository.objects.filter(
+            repo_id=self.repo_id
+        ).first()
+
+        if github_repository:
+            return github_repository.user
+
+        return None
+
+    def queue_refresh_commits(self, *, ref):
+        from .jobs import refresh_commits_job
+
+        refresh_commits_job.delay(repository=self, branch_name=ref)
+
+    @transaction.atomic
+    def add_commits(self, *, commits, ref):
+        matching_tasks = Task.objects.filter(branch_name=ref, project__repository=self)
+
+        for task in matching_tasks:
```

Comment:
Again, this _should_ be a single task... but it doesn't hurt.

Reply:
Yep.